### PR TITLE
random fixes

### LINKED ISSUE #112: Add Microsoft SQL Server database support

### PR DESCRIPTION
**Pull request by amreo**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
-    "AWR": 30,
-    "parallelizeRequests": true,
-    "verbose": false,
-    "features": {
-        "databases": {
-            "enabled": true
+    "Hostname": "default",
+    "Environment": "TST",
+    "Location": "Germany",
+    "DataserviceURL": "http://dev.ercole.io/data/hosts",
```
Comment (cirelli94) [inline config.json:5]: DataserviceURL should be without `/hosts`

Comment (cirelli94) [inline config.json:5]: DataserviceURL should be without `/hosts`

File: main.go [hunk 2]
```diff
@@ -91,11 +90,9 @@ func sendData(data *model.HostData, configuration config.Configuration, log logg
 		}
 	}
 
-	req, err := http.NewRequest("POST", configuration.Serverurl, bytes.NewReader(dataBytes))
+	req, err := http.NewRequest("POST", configuration.DataserviceURL, bytes.NewReader(dataBytes))
```
Comment (cirelli94) [inline main.go:93]: .. and we should add `/hosts` here!

File: config.json (hunk 1 above)
Comment (cirelli94) [reply config.json:5]: And maybe DataService?

**Comment by cirelli94** [comment]
Does this PR fix #112 ???

File: config.json (hunk 1 above)
Comment (amreo) [reply config.json]: ...no...

**Pull request merged**